mitmproxy/addons/core.py

```diff
@@ -50,18 +50,19 @@ def configure(self, updated):
                     )
 
     @command.command("set")
-    def set(self, option: str, value: str = "") -> None:
+    def set(self, option: str, *value: str) -> None:
         """
             Set an option. When the value is omitted, booleans are set to true,
             strings and integers are set to None (if permitted), and sequences
             are emptied. Boolean values can be true, false or toggle.
             Multiple values are concatenated with a single space.
-            Sequences are set using multiple invocations to set for
-            the same option.
         """
-        strspec = f"{option}={value}"
+        if value:
+            specs = [f"{option}={v}" for v in value]
+        else:
+            specs = [option]
```

Comment:
Shouldn't this be `f"{option}"`?

Reply:
`option` can only be a str, or am I missing something?

Reply:
right, I thought for a moment that `option` is of Option type, not a name.
I think such a confusion could be avoided if we have a method mentioned in https://github.com/mitmproxy/mitmproxy/pull/5067/files#r786017760